packages/element-web-module-api/src/loader.ts

```diff
@@ -0,0 +1,46 @@
+/*
+Copyright 2025 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+Please see LICENSE files in the repository root for full details.
+*/
+
+import { satisfies } from "semver";
+
+import { Api, isModule, Module, ModuleExport } from "./api";
+
+export class ModuleIncompatibleError extends Error {
+    public constructor(pluginVersion: string) {
+        super(`Plugin version ${pluginVersion} is incompatible with engine version ${__VERSION__}`);
+    }
+}
+
+export class ModuleLoader {
+    public constructor(private api: Api) {}
+
+    #modules: Module[] = [];
```

Comment:
This code uses both js private properties and typescript `private`. What is the expectation here?

Reply:
Have consolidated to ts variant